testsuite/MDAnalysisTests/utility/pylibxdr.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+import numpy as np
+
+
+def unpack_array(xdr, shape):
+    arr = np.array([xdr.unpack_float() for _ in range(np.prod(shape))])
+    return arr.reshape(shape)
+
+
+# Poor mans namespace
+class XTC:
```

Comment:
Isn't that a comment already? If you have suggestions for the naming and where a good place for this would be I'm all ears. I think the names I have chosen could be improved.